internal/commands/config.go

```diff
@@ -318,17 +317,21 @@ func (c Config) Confirm(context string) error {
 		return nil
 	}
 	inst, err := readline.NewEx(&readline.Config{
-		Prompt: "Do you want to continue [y/n]: ",
-		Stdin:  c.stdin,
-		Stdout: c.stdout,
-		Stderr: c.stderr,
+		Prompt:              "Do you want to continue [y/n]: ",
+		Stdin:               c.stdin,
+		Stdout:              c.stdout,
+		Stderr:              c.stderr,
+		ForceUseInteractive: true,
 	})
 	if err != nil {
 		return err
 	}
 	for {
 		s, err := inst.Readline()
 		if err != nil {
+			if err == io.EOF {
+				return errors.New("failed to get user confirmation")
+			}
```

Comment:
can we tell user about the `QBEC_DISABLE_PROMPTS` option here, might be useful?

Reply:
I think those are not related always. There could be other reasons for for EOF. We don't want to encourage users to add QBEC_DISABLE_PROMPTS. I think CI would probably be a good place to use this flag. Docs have been updated.